### ISSUE TITLE
Installer error when installing required packages and dependencies (CentOS 7 & RHEL 7)

### ISSUE THREAD

**Issue by StrangeRanger** [CodeBullet-Community/BulletBot#42]
**Describe the bug**
When using the installer on CentOS 7 and RHEL 7, an error occurs when trying to install the required packages and dependencies.
Even though this error occurs, upon returning to the installer menu, it says that the required packages and dependencies are installed. Though some are, not all of them are installed.

CentOS 7
![Screen Shot 2019-11-06 at 7 51 21 PM](https://user-images.githubusercontent.com/30395560/68358997-34114a00-00cf-11ea-879c-df86bec7e5f6.png)

RHEL 7
![Screen Shot 2019-11-06 at 8 16 38 PM](https://user-images.githubusercontent.com/30395560/68359834-67a1a380-00d2-11ea-9561-4a7d85c5d513.png)

**Setup (please complete the following information):**
 - OS: CentOS 7 & RHEL 7
 - Node.js Version: 12.x.x
 - MongoDB Version: 4.2.1

**Temporary fix**
1. Exit the installer
2. Make your way to `/home/bulletbot`
3. Execute `sudo bash linux-master-installer.sh`
4. Choose option 4 in the installer menu
5. Press [Enter]
6. It should install everything

**IMPORTANT**
I've isolated this bug to CentOS 7 and RHEL 7 servers that are hosted by AWS. This does not occur on servers hosted, for example, by DigitalOcean.

**Comment by StrangeRanger** [comment]
Isolated the bug to AWS based servers

**Comment by StrangeRanger** [comment]
This issue is affects any server (that uses CentOS 7 or RHEL 7), hosted by AWS. Therefore this issue is out of our control. We will look into a solution, though there might not be one, nor might one be worth creating, due to the issue being a one-time thing.

**Comment by StrangeRanger** [comment]
This issue is affects any server (that uses CentOS 7 or RHEL 7), hosted by AWS. Therefore this issue is out of our control.

**Comment by StrangeRanger** [comment]
After more testing, I was able to find out the cause of the error. It had to do with the permission of `bulletbot's` home directory. Usually, when `/home/bulletbot` is created, it is given the permissions `755`, but on AWS based servers that host CentOS 7 and RHEL 7, the permissions given to `/home/bulletbot` are 700, which caused the error. A fix has been implemented, which will be in the next release.
